source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+  font-size: 1rem; // 16px
```

Comment:
This property is unnecessary. The element should automatically inherit the base font size.